### PR TITLE
fix: remove shuf command 

### PR DESCRIPTION
**Pull request by FionnD**
Removed shuf command since it doesn't exist on Mac. 

Alex, please let me know if you think the new logic in the data file is okay!

### CODE REVIEW

File: wikipedia-sentences/README.md [hunk 1]
```diff
@@ -46,6 +46,8 @@ To index the [full dataset](https://www.kaggle.com/mikeortman/wikipedia-sentence
 5. Delete the old index: `rm -rf workspace`
 6. Index your new dataset: `python app.py -t index`
 
+If you are using a subset of the data (less then 30,000 documents) we recommend you shuffle the data to prevent only indexing a specific letter of the alphabet. On linux on you can use the `shuf` command. 
+
```
Comment (alexcg1) [inline wikipedia-sentences/README.md:50]: ## Rewrite

(Let me know if it's too wordy)

If you are using a subset of the data (less then 30,000 documents) we recommend you shuffle the data. This is because the input file is ordered alphabetically, and Jina indexes from the top down. So without shuffling, your index may contain unrepresentative data, like this:

```
0.000123, which corresponds to a distance of 705 Mly, or 216 Mpc.
000webhost is a free web hosting service, operated by Hostinger.
0010x0010 is a Dutch-born audiovisual artist, currently living in Los Angeles.
0-0-1-3 is an alcohol abuse prevention program developed in 2004 at Francis E. Warren Air Force Base based on research by the National Institute on Alcohol Abuse and Alcoholism regarding binge drinking in college students.
0.01 is the debut studio album of H3llb3nt, released on February 20, 1996 by Fifth Colvmn Records.
001 of 3 February 1997, which was signed between the Government of the Republic of Rwanda, and FAPADER.
003230 is a South Korean food manufacturer.
```

On Linux, you can shuffle using the [`shuf` command](https://linuxhint.com/bash_shuf_command/):

```bash
shuf input.txt > input.txt
```

To shuffle a file on macos, please read [this post](https://apple.stackexchange.com/questions/142860/install-shuf-on-os-x/195387).

**Review by alexcg1 — CHANGES_REQUESTED** [review]

**Comment by alexcg1** [comment]
I think in the medium-long run it would make sense to retire this data file altogether since it's so broad (as I've discussed on Slack).

I've been testing alternatives and will keep everyone posted

File: wikipedia-sentences/README.md (hunk 1 above)
Comment (FionnD) [reply wikipedia-sentences/README.md:68]: Way better! Accepted and added.

**Review by alexcg1 — APPROVED** [review]
LGTM :+1:
